.github/actions/expose-pythons/action.yml

```diff
@@ -0,0 +1,22 @@
+name: Expose Pythons
```

Comment:
This is lifted from here: https://github.com/pantsbuild/pants-jupyter-plugin/blob/main/.github/actions/expose-pythons/action.yml

It's well tested. A follow-up could be to make an actions repo for this that both projects use.